version.py

```diff
@@ -26,4 +26,4 @@
 #   historical commits sharing the same major version, in which case a
 #   minor version bump suffices.
 
-PROVISION_VERSION = '67.0'
+PROVISION_VERSION = '67.1'
```

Comment:
This should have been a major bump for the dependency removal. (However, it’s becoming clear that we need a better system for maintaining provision bisectability than manual review…)

Reply:
I've for a long time been of the opinion that we should make the obvious venv/npm parts of PROVISION_VERSION maintenance be done by some sort of comparison of the requirements files, since it would allow us to remove 95% of cases where we need to update `PROVISION_VERSION` because of this sort of rule...